### DISCUSSION TITLE
how do I download the mod!!!!!!

### DISCUSSION THREAD

**Discussion by Hemakshsharma** [MartinTheDragon/Nuclear-Tech-Mod-Remake#53] [General]
how do I download the mod!!!!!! show me the steps to download it

**Comment by 70000hp** [comment]
![image](https://user-images.githubusercontent.com/105080577/224494195-ff74f39d-799d-4217-a056-bd2984e928d8.png)
![image](https://user-images.githubusercontent.com/105080577/224494229-32b61c18-795c-4268-9034-47b841cf7a86.png)
![image](https://user-images.githubusercontent.com/105080577/224494280-ef39faa6-ea1f-4404-9f5a-678af4766292.png)

**Reply by Hemakshsharma** [reply]
thank you
